SwiftUIBasics/Views/SignUpView.swift

```diff
@@ -18,16 +18,25 @@ class SignUpViewModel: ObservableObject {
     @Published var isValidUsernameLength: Bool = false
```

Comment:
the binding name does not match the logic, because you are validating email in username variables